### DISCUSSION TITLE
Hey please invite me in your Organisation I would Seeing your organisation since 7+ months.

### DISCUSSION THREAD

**Discussion by mdazfar2** [AccessibleForAll/Support#659] [General]
I have a different innovatives ideas and I am also a professional DevOps Practitioner.

**Comment by ronknight** [comment]
Hi,

This is to inform everyone that all people added in the organization are voluteer. It takes time to anwer issues. We are going to get back to all issues as soon as possible. Thank you for your patience.

**Comment by ronknight** [comment]
> I have a different innovatives ideas and I am also a professional DevOps Practitioner.

Please fill up form and accept invitation. [https://github.com/AccessibleForAll/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization](https://github.com/AccessibleForAll/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization)

**Reply by ronknight** [reply]
![image](https://github.com/AccessibleForAll/Support/assets/7566286/8ed209f3-d6c8-4b99-ba00-99262da35c41)

**Comment by sandipanrakshit34** [comment]
hey please give me a start on this repository and help me to achieve the badge:

https://github.com/sandipanrakshit34/sandipanrakshit34
